source/includes/steps-enable-authentication-in-shardcluster-nodowntime-auth-mongos.yaml

```diff
@@ -0,0 +1,53 @@
+stepnum: 1
+level: 4
+title: "Remove ``transitionToAuth`` from the ``mongos`` configuration file"
+ref: modify-config-files
+pre: |
+
+  Remove the :setting:`security.transitionToAuth` key and its value from the
+  configuration file created during this tutorial.
+  
+  .. code-block:: yaml
+
+     security:
+        keyFile: <path-to-keyfile>
+
+---
+stepnum: 2
+level: 4
+ref: restart-mongos
+title: "Restart the ``mongos`` with the updated configuration file."
+pre: |
+
+  Connect to the :program:`mongos` and use the :method:`db.shutdownServer()`
+  method against the ``admin`` database to safely shut down the
+  :program:`mongos`.
+  
+  Restart one :program:`mongos` at a time to ensure that clients
+  can connect to the sharded cluster. If your cluster has only one
+  :program:`mongos`, this step results in downtime while the :program:`mongos`
+  is offline.
+  
+  .. code-block:: javascript
+  
+     db.getSiblingDB("admin").shutdownServer()
+  
+  You can then restart the :program:`mongos` specifying the path to the updated
+  config file using :option:`--config`.
+  
+  .. example::
+
+     The following operation restarts the :program:`mongos` specifying
+     the updated configuration file, here named ``mongos-secure.conf``.
+     The ``<path>`` represents the system path to the folder containing
+     the configuration file.
+
+     .. code-block:: shell
+     
+        mongos --config <path>/mongos-secure.conf
+
+  .. note::
```

Comment:
This note seems like it belongs with the example.  in which case, I might not make it a note.